spec/redux/actionCreators-test.js

```diff
@@ -6,137 +6,104 @@ import thunk from 'redux-thunk';
 import Immutable from 'immutable';
 
 describe("The actionCreators'", () => {
-
   const middlewares = [thunk];
   const mockStore = configureStore(middlewares);
+  var store;
+
+  beforeEach(() => {
+    store = mockStore({});
+  });
```

Comment:
Before, we were initializing store in every describe block and setting it to mockStore in nearly every it() block in our tests. Now, we simply set store to mockStore before each test.